src/main/java/OrderDelocalized/Customer.java

```diff
@@ -11,46 +11,53 @@ public class Customer {
     private List<Order> orderHistory;
 
     public Customer(String name, String email, boolean isMember) {
-
+        this.name = name;
+        this.email = email;
+        this.isMember = isMember;
+        this.openOrders = new ArrayList<>();
+        this.orderHistory = new ArrayList<>();
```

Comment:
Should we want the declarations above to be in the commit too? or leave the initialization out of the commit?